test/sim/misc/accuracy.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const assert = require('./../../assert');
+const common = require('./../../common');
+
+let battle;
+
+describe("Accuracy", function () {
+	afterEach(function () {
+		battle.destroy();
+	});
+
+	it(`should round half down when applying a modifier`, function () {
+		battle = common.createBattle([
+			[{species: 'Butterfree', ability: 'compoundeyes', moves: ['sleeppowder']}],
+			[{species: 'Beldum', moves: ['poltergeist']}],
+		]);
+
+		battle.onEvent('Accuracy', battle.format, function (accuracy) {
+			assert.equal(accuracy, 98, 'CompoundEyes Sleep Powder should be 98% accurate');
+		});
+
+		battle.makeChoices();
+
+		battle = common.createBattle([
+			[{species: 'Butterfree', ability: 'victorystar', moves: ['fireblast']}],
+			[{species: 'Regirock', moves: ['sleeptalk']}],
+		]);
+
+		battle.onEvent('Accuracy', battle.format, function (accuracy) {
+			assert.equal(accuracy, 94, 'Victory Star Fire Blast should be 93% accurate');
```

Comment:
I'm guessing the assertion is correct but the description isn't? Ditto the Wide Lens one below.

Reply:
Yes, I'm bad